nexusutils/dataconverter/template.py

```diff
@@ -165,3 +165,15 @@ def add_entry(self, entry_name):
         template = Template(self)
         template.rename_entry("entry", entry_name, False)
         self.update(template)
+
+    def __delitem__(self, key):
```

Comment:
This is not used anymore. Should it be kept or not?

Reply:
My opinion is on behalf of keeping this.

Reply:
Fine with me. I just wanted to make sure that it wasn't just forgotten